backend/tests/core/Entities/BuildingPredominateUseHelper.cs

```diff
@@ -41,18 +41,20 @@ public static List<Entity.BuildingPredominateUse> CreateDefaultBuildingPredomina
                 new Entity.BuildingPredominateUse(1, "Research & Development Facility") { RowVersion = new byte[] { 12, 13, 14 } },
                 new Entity.BuildingPredominateUse(2, "Residential Detached") { RowVersion = new byte[] { 12, 13, 14 } },
                 new Entity.BuildingPredominateUse(3, "Residential Multi") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(4, "Senior Housing (Assisted Living / Skilled Nursing)") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(5, "Shelters / Orphanages / Children’s Homes / Halfway Homes") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(6, "Social Assistance Housing") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(7, "Storage") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(8, "Storage Vehicle") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(9, "Trailer Office") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(10, "Trailer Other") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(11, "Training Center") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(12, "Transportation (Airport / Rail / Bus station)") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(13, "University / Collect") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(14, "Warehouse") { RowVersion = new byte[] { 12, 13, 14 } },
-                new Entity.BuildingPredominateUse(15, "Weigh Station") { RowVersion = new byte[] { 12, 13, 14 } }
+                new Entity.BuildingPredominateUse(4, "Retail") { RowVersion = new byte[] { 12, 13, 14 } },
```

Comment:
"Retail" was missing so all the numbers were off, not sure if this affects anything too much